roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/mocks/RouteGenerator.java

```diff
@@ -48,170 +50,227 @@ public IRoute getRoute(RouteType type) {
                 return getRouteWithOneStreetSectionAndOneOneWayStreet();
             case TWO_ONEWAYSTREETS:
                 return getRouteWithTwoOneWayStreets();
+            case TWO_STREETSECTIONS_ONE_CAR:
+                return getRouteWithTwoStreetSectionsOneCar();
         }
         return null;
     }
 
     private void initializeRoutes(RoundaboutSimulationModel model) {
-
         // INITIALIZE ROUTE WITH TWO STREETSECTIONS
+        initRouteWithTwoStreetSections(model);
+
+        // INITIALIZE ROUTE WITH ONEWAYSTREET AND STREETSECTION
```

Comment:
those comments are not necessary, as the method name itself is saying that, same for the other comments below